tests/unit/components/grapher/Dot.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { Wrapper, createLocalVue, mount } from "@vue/test-utils";
+import Vue, { VueConstructor } from "vue";
+import Vuex, { Store } from "vuex";
+import Grapher from "@/components/grapher/Grapher.vue";
+import Dot from "@/components/grapher/Dot.vue";
+import { CalChartState, generateStore } from "@/store";
+import Show from "@/models/Show";
+import StuntSheet from "@/models/StuntSheet";
+import StuntSheetDot from "@/models/StuntSheetDot";
+import DotAppearance from "@/models/DotAppearance";
+
+jest.mock("svg-pan-zoom", () => {
+  return {
+    __esModule: true,
+    default: jest.fn().mockReturnValue({ resize: jest.fn() }),
+  };
+});
+
+describe("components/grapher/Dot.vue", () => {
+  let localVue: VueConstructor<Vue>;
+  let store: Store<CalChartState>;
+  let wrapper: Wrapper<Vue>;
+  beforeAll(() => {
+    localVue = createLocalVue();
+    localVue.use(Vuex);
+  });
+  describe("stuntSheetDot", () => {
+    it("renders the basic dot", () => {
+      const stuntSheetDots = [];
+      stuntSheetDots.push(
+        new StuntSheetDot({
+          x: 10,
+          y: 10,
+          dotTypeIndex: 0,
+        })
+      );
+      store = generateStore({
+        show: new Show({
+          stuntSheets: [new StuntSheet({ stuntSheetDots })],
+        }),
+      });
+      wrapper = mount(Grapher, {
+        store,
+        localVue,
+      });
+      expect(wrapper.findComponent(Dot).props("dotTypeIndex")).toBe(0);
+      expect(wrapper.find('[data-test="dot--circle"]').exists()).toBeTruthy();
+      expect(wrapper.find('[data-test="dot--circle"]').attributes("fill")).toBe(
+        "black"
+      );
+      expect(
+        wrapper.find('[data-test="dot--circle"]').attributes("fill-opacity")
+      ).toBe("1");
+      expect(
+        wrapper.find('[data-test="dot--circle"]').attributes("stroke")
+      ).toBe("black");
+      expect(wrapper.find('[data-test="dot--fslash"]').exists()).toBeFalsy();
+      expect(wrapper.find('[data-test="dot--bslash"]').exists()).toBeFalsy();
+    });
+    it("renders a modified dot", () => {
+      const stuntSheetDots = [];
+      stuntSheetDots.push(
+        new StuntSheetDot({
+          x: 10,
+          y: 10,
+          dotTypeIndex: 0,
+        })
+      );
+      const dotAppearances = [];
+      dotAppearances.push(
+        new DotAppearance({
+          filled: false,
+          fill: "red",
+          color: "blue",
+          fwSlash: true,
+          bwSlash: true,
+        })
+      );
+      store = generateStore({
+        show: new Show({
+          stuntSheets: [
+            new StuntSheet({
+              stuntSheetDots: stuntSheetDots,
+              dotAppearances: dotAppearances,
+            }),
+          ],
+        }),
+      });
+      wrapper = mount(Grapher, {
+        store,
+        localVue,
+      });
+      expect(wrapper.findComponent(Dot).props("dotTypeIndex")).toBe(0);
```

Comment:
I don't necessarily recommend doing anything about this right now, but these tests are getting long. We may want to consider creating a utility library that has some functions to help compose these without creating these bulky test cases.

Reply:
Yeah, I agree. I haven't said it explicitly with the group but I want to shift to developing without verbose testing... I'll bring it up at the next meeting. While we're still in the early stages of this redesign, we will have a faster velocity without worrying about tests. It gives us more flexibility to change our mind on the high level technical designs.